app/bundles/CoreBundle/ErrorHandler/ErrorHandler.php

```diff
@@ -290,6 +292,10 @@ public static function prepareExceptionForOutput($exception)
             $trace             = $exception->getTrace();
             $context           = (method_exists($exception, 'getContext')) ? $exception->getContext() : [];
 
+            if ($originalException instanceof MessageOnlyErrorHandlerException) {
+                $file = $trace = $context = $line = '';
+            }
```

Comment:
What is this for?

Reply:
to ensure the message doesn't look like this, as the file, line and stracktrace aren't relevant for this kind of message IMO

<img width="930" alt="Screenshot 2024-01-12 at 19 21 31" src="https://github.com/mautic/mautic/assets/3983285/5a1c6624-3709-4e62-a7b6-0b59e051ca24">

Reply:
This will show up only if you have env=dev. In production there will be only the "Uh oh" message, right?

Reply:
no, as it's an error before Mautic is even "booted" it's this kind over error, also in prod mode

Reply:
that makes me wonder: why does the stacktrace show up in the first place in prod mode? I'll check again, as it shouldn't by default